rakelib/lint.rake

```diff
@@ -0,0 +1,22 @@
+namespace :lint do
+  desc 'Install swiftlint'
+  task :install do |task|
+    swiftlint = `which swiftlint`
```

Comment:
Weird, I actually changed this because it kept failing for me on undefined something 'success'